modules/installation-vsphere-config-yaml.adoc

```diff
@@ -26,97 +29,62 @@ apiVersion: v1
 baseDomain: example.com <1>
 compute: <2>
 - architecture: amd64
-  hyperthreading: Enabled <3>
   name: <worker_node>
   platform: {}
-  replicas: 0 <4>
+  replicas: 0 <3>
 controlPlane: <2>
   architecture: amd64
-  hyperthreading: Enabled <3>
   name: <parent_node>
   platform: {}
-  replicas: 3 <5>
+  replicas: 3 <4>
 metadata:
   creationTimestamp: null
-  name: test <6>
+  name: test <5>
 networking:
 ---
 platform:
   vsphere:
-    failureDomains: <7>
+    failureDomains: <6>
     - name: <failure_domain_name>
       region: <default_region_name>
       server: <fully_qualified_domain_name>
       topology:
         computeCluster: "/<datacenter>/host/<cluster>"
-        datacenter: <datacenter> <8>
-        datastore: "/<datacenter>/datastore/<datastore>" <9>
+        datacenter: <datacenter> <7>
+        datastore: "/<datacenter>/datastore/<datastore>" <8>
         networks:
         - <VM_Network_name>
-        resourcePool: "/<datacenter>/host/<cluster>/Resources/<resourcePool>" <10>
-        folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <11>
+        resourcePool: "/<datacenter>/host/<cluster>/Resources/<resourcePool>" <9>
+        folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <10>
       zone: <default_zone_name>
     vcenters:
     - datacenters:
       - <datacenter>
-      password: <password> <12>
+      password: <password> <11>
       port: 443
-      server: <fully_qualified_domain_name> <13>
+      server: <fully_qualified_domain_name> <12>
       user: administrator@vsphere.local
-    diskType: thin <14>
-ifndef::restricted[]
-ifndef::openshift-origin[]
-fips: false <15>
-endif::openshift-origin[]
-ifndef::openshift-origin[]
-pullSecret: '{"auths": ...}' <16>
-endif::openshift-origin[]
-ifdef::openshift-origin[]
```

Comment:
Correct me if I'm wrong, but it looks like a bunch of content got removed here. Lines 66-74, as well as most of lines 79-107. why was it removed?

Reply:
A lot of these lines were repeats that stemmed around the `openshift-origin` variable, which seems defined at a higher level? I removed this variable from the docs and, instead, created more direct variables that target specific modules. I had the luxury of restructuring this file as only 3 vSphere modules are impacted.